src/Type/Definition/TypeInterface.php

```diff
@@ -2,15 +2,10 @@
 
 namespace Digia\GraphQL\Type\Definition;
 
-interface TypeInterface
+interface TypeInterface extends DefinitionInterface
 {
     /**
      * @return string|null
      */
     public function getName(): ?string;
-
-    /**
-     * @return string|null
-     */
-    public function getDescription(): ?string;
```

Comment:
This was actually a bug, `ListType` and `NonNullType` cannot have a description.